docs/insights/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Insights
+description: Insights gives you better visibility into your company's DevOps performance.
+position: 100
+---
+
+![The Overview page of Insights Reports](images/overview.png)
+
+DevOps insights (early access) in Octopus gives you better visibility into your company’s DevOps performance by surfacing the four key DORA metrics, so you can make more informed decisions on where to improve and celebrate your results. 
+
+There are two levels available for DevOps Insights: Project level insights are available to all customers, and Space level insights are available to Enterprise customers. 
```

Comment:
Just a suggestion, I think the overview page is better after the first paragraph.

```suggestion
DevOps insights (early access) in Octopus gives you better visibility into your company’s DevOps performance by surfacing the four key DORA metrics, so you can make more informed decisions on where to improve and celebrate your results. 

![The Overview page of Insights Reports](images/overview.png)

Two levels are available for DevOps Insights:
1. Project-level insights, available to all customers.
2. Space-level insights, available to customers with an [enterprise license](https://octopus.com/enterprise).
```